src/components/Account/Account.tsx

```diff
@@ -6,6 +6,7 @@ import DOMPurify from 'dompurify';
 import { useMutation } from 'react-query';
 
 import styles from './Account.module.scss';
+import '../../styles/accessibility.scss';
```

Comment:
This important isn't needed, since you're importing it from the scss file as well.

Reply:
Thanks!